src/Relations/Concerns/InteractsWithPivotTable.php

```diff
@@ -194,9 +194,11 @@ protected function restoreUsingCustomClass($ids)
     protected function getCurrentlyAttachedPivots()
     {
         return $this->newPivotQueryWithoutTrashed()->get()->map(function ($record) {
-            $class = $this->using ? $this->using : Pivot::class;
+            $class = $this->using ?: Pivot::class;
```

Comment:
Don't need to change thus line. In this case you will have empty $class if $this->using is empty.

Reply:
I've just copied it from the original Laravel function, they changed it there as well:

https://github.com/laravel/framework/commit/c78b3b4ef50f1b65aabcd2b958646993ff34548b#diff-5603d4d15ca47b196de2c3f2c70590b20cb523a393228fc7f294a2c6a54c443c

Reply:
By the way, I don't think it changes any functionality as ?: is just a shorter notation for the same statement. According to the PHP documentation:

"The expression (expr1) ? (expr2) : (expr3) evaluates to expr2 if expr1 evaluates to true, and expr3 if expr1 evaluates to false.

It is possible to leave out the middle part of the ternary operator. Expression expr1 ?: expr3 returns expr1 if expr1 evaluates to true, and expr3 otherwise. "

https://www.php.net/manual/en/language.operators.comparison.php